cwlformat/formatter.py

```diff
@@ -3,7 +3,11 @@
 from typing import Union
 import sys
 import pathlib
-import importlib.resources as pkg_resources
+try:
+    from importlib.resources import read_text
+except ImportError:
+    # Python 2.x backport
```

Comment:
Perhaps change this to refer to the exact Python range. I would suggest 3.6 <= Py

Reply:
Changed comment in d37a22fa90da12529445be5d7de4d61f2f9d5bf9  to:
```
# Python 3.6 fallback: https://importlib-resources.readthedocs.io/en/latest/
```